dnase/callsnpsMerge.sh

```diff
@@ -29,6 +29,7 @@ analysisType=${2}
 sampleOutdir=${3}
 sampleAnnotation=${4}
 src=${5}
+analyzedReads=${6}
```

Comment:
can we check whether the bedgraph files are empty rather than adding a new argument?